build/build.sh

```diff
@@ -79,11 +79,23 @@ if $OSX; then
 else
   OSNAME="linux"
 fi
-DAEMON_URL="$(cat "$BUILD_DIR/DAEMON_URL" | sed "s/OSNAME/${OSNAME}/")"
+DAEMON_VER=$(cat "$BUILD_DIR/DAEMON_VER")
+DAEMON_URL="https://github.com/lbryio/lbry/releases/download/v${DAEMON_VER}/lbrynet-daemon-v${DAEMON_VER}-${OSNAME}.zip"
 wget --quiet "$DAEMON_URL" -O "$BUILD_DIR/daemon.zip"
 unzip "$BUILD_DIR/daemon.zip" -d "$ROOT/app/dist/"
 rm "$BUILD_DIR/daemon.zip"
 
+
+###################
+#   Config file   #
+###################
+
+cat << EOF > "$ROOT/app/config.json"
+{
```

Comment:
The reason I left it as separate files is that they're conceptually different things. `config.json` is the configuration file for the *app*, which is *generated* by the build process. Some parts of `config.json` might even be dynamically generated eventually.

There would definitely be a case for making a `build_config.json` that in turn builds this file. We could also put all of the other build constants in there, like the URL format.

npm does let you put configuration in package.json. Some Node projects use that, some use a `config.json` file. `config.json` seemed more simpler and explicit, especially because we may end up needing to dynamically generate the file at build time.

On the other hand, I just looked into it some more and it turns out NPM lets you programmatically modify the configuration in package.json via `npm config`. It would be really nice if we could do that in the build script. I'll look into it.